portal-web/docroot/html/taglib/ui/input_permissions/page.jsp

```diff
@@ -119,11 +119,15 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 					<option <%= inputPermissionsViewRole.equals(RoleConstants.OWNER) ? "selected=\"selected\"" : "" %> value="<%= RoleConstants.OWNER %>"><liferay-ui:message key="owner" /></option>
 				</select>
 
-				<div class="mt-3 <%= inputPermissionsShowOptions ? "hide" : "" %>" id="<%= uniqueNamespace %>inputPermissionsShowOptionsLink" style="margin-bottom: 60px;">
-					<a class="btn btn-secondary btn-sm mt-3" href="javascript:<%= uniqueNamespace %>inputPermissionsShowOptions();" role="button"><liferay-ui:message key="more-options" /></a> <liferay-ui:icon-help message="input-permissions-more-options-help" />
+				<div>
+					<button aria-controls="<%= uniqueNamespace %>inputPermissionsTable" aria-expanded="<%= inputPermissionsShowOptions %>" class="btn btn-secondary btn-sm mt-3" id="<%= uniqueNamespace %>inputPermissionsOptionsButton" onclick="<%= uniqueNamespace %>inputPermissionsToggle();" type="button" value="<%= LanguageUtil.get(request, "more-options") %>">
+						<liferay-ui:message key="more-options" />
+					</button>
+
+					<span class="mt-3 <%= inputPermissionsShowOptions ? "hide" : "" %>" id="<%= uniqueNamespace %>inputPermissionsShowOptionsLink" style="margin-bottom: 60px;">
```

Comment:
Can we change this `id` please? It is confusing because it is not a link, maybe `inputPermissionsShowOptionsHelp`?
I don't even like the `inputPermissionsShowOptions` prefix, but that would mean lot of changes, so for now we can leave it as it is.